microservices-connector/config/samples/chatQnA_xeon.yaml

```diff
@@ -33,6 +34,7 @@ spec:
           serviceName: retriever-svc
           config:
             endpoint: /v1/retrieval
+            REDIS_URL: VectorDB
```

Comment:
same here, use `redis-vector-db`, in this way we can support multiple endpoint for switch scenario

Reply:
In the multiple endpoint case, is it better to use different step name, like llm-big, llm-small?